lib/portage/util/_async/PipeReaderBlockingIO.py

```diff
@@ -15,10 +15,7 @@ class PipeReaderBlockingIO(AbstractPollTask):
     """
     Reads output from one or more files and saves it in memory, for
     retrieval via the getvalue() method. This is driven by a thread
-    for each input file, in order to support blocking IO.  This may
-    be useful for using threads to handle blocking IO with Jython,
-    since Jython lacks the fcntl module which is needed for
-    non-blocking IO (see http://bugs.jython.org/issue1074).
+    for each input file, in order to support blocking IO.
```

Comment:
I would again keep a historical comment so we can re-evaluate if the whole thing is needed in future?

Reply:
Alright, I can leave the historical comments in.

Reply:
Actually, I'll just remove these files from the commit since it'd be just a comment change.

Reply:
I think it might be worth leaving the historical comments in but rephrasing to make clear that Jython is longer a concern.

Reply:
Ah, I can do that.